scripts/check-env.sh

```diff
@@ -3,13 +3,37 @@
 # Check the environment for dependencies and configuration
 #
 
+# check appropriate package installer to verify the packages
+YUM_CMD=$(command -v yum)
```

Comment:
On newer systems `yum` is replaced by `dnf`. So maybe you should check for `rpm` instead. You're executing `rpm` commands below, so it would be good to verify that the command exists in any case.